starter/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.google.cloud.trace.adapters.zipkin</groupId>
+        <artifactId>parent</artifactId>
+        <version>0.2.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>starter</artifactId>
```

Comment:
ps by support burden I meant people coming into gitter who found the starter and started playing with it but don't know anything about tracing or what they need it for. requests for support on how to get eclipse working because they feel they need a custom server to start zipkin.. or blogs that explain how to do basic things already done by the default server, which propagate misunderstanding on how to configure things or that basic things don't work out-of-box. things like this